python/tvm/relax/frontend/stablehlo/stablehlo_translator.py

```diff
@@ -0,0 +1,417 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# pylint: disable=invalid-name, inconsistent-return-statements, unidiomatic-typecheck
+# pylint: disable=import-outside-toplevel, unused-argument, no-member
+"""StableHLO frontend of Relax."""
+from typing import Callable, Dict, List, Tuple, Union, Any
+import numpy as np
+
+from jaxlib import mlir
+from jaxlib.mlir.dialects import stablehlo
+
+import tvm
+from tvm import relax, tir
+
+
+class StableHLOImporter:
```

Comment:
As we may have more importers, should we create a base class to reuse some utils? cc @spectrometerHBH @MasterJH5574

Reply:
That's a good idea. We can certainly proceed with it. For now in this pr, we may consider this as a todo, as the currently available shared helper functions are somewhat limited.